pkg/controller/template/render.go

```diff
@@ -392,8 +393,8 @@ func skipMissing(key string) (interface{}, error) {
 }
 
 func cloudProvider(cfg RenderConfig) (interface{}, error) {
-	if cfg.Infra.Status.PlatformStatus != nil {
-		external, err := cloudprovider.IsCloudProviderExternal(cfg.Infra.Status.PlatformStatus, cfg.FeatureGate)
+	if cfg.Infra.Status.PlatformStatus != nil && cfg.FeatureGateAccess != nil {
```

Comment:
Not all render configs will have the FGAccess, assume the featuregates are off in this case (shouldn't actually matter)